git/move_milestones.go

```diff
@@ -0,0 +1,87 @@
+package git
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/google/go-github/github"
+)
+
+func MoveMilestone(ghClient *github.Client, repo string, oldMilestoneName string, newMilestoneName string, includeClosed bool) error {
+	is := ghClient.Issues
+	milestones, _, err := is.ListMilestones("deis", repo, &github.MilestoneListOptions{})
+	if err != nil {
+		return err
+	}
+	var oldMilestone *github.Milestone
+	for _, milestone := range milestones {
+		if *milestone.Title == oldMilestoneName {
+			oldMilestone = &milestone
+			break
+		}
+	}
+	if oldMilestone == nil {
+		return fmt.Errorf("milestone %s not found", oldMilestoneName)
+	}
+	var newMilestone *github.Milestone
+	for _, milestone := range milestones {
+		if *milestone.Title == newMilestoneName {
+			newMilestone = &milestone
+			break
+		}
+	}
+	if newMilestone == nil {
+		return fmt.Errorf("milestone %s not found", newMilestoneName)
+	}
```

Comment:
Just a DRY nit here (spanning lines 16-35 above).  Might consider breaking out the duplicated logic here into a helper function `func GetMilestone(milestoneName string) (*github.Milestone, error)` or some such that could be called with both `oldMilestoneName` and `newMilestoneName` above, returning the `*github.Milestone` if exists in GH or an error if not.